biztalk/install-and-config-guides/hardware-and-software-requirements-for-biztalk-server-2020.md

```diff
@@ -34,19 +34,21 @@ The following table lists the minimum hardware requirements for your BizTalk Ser
+| Microsoft Visual C++ 2015-2019 Redistributable Package |                                The x86 and x64 versions are required. Download at [Visual C++ 2015-2019 Redistributable Package x86](https://aka.ms/vs/16/release/VC_redist.x86.exe) and [Visual C++ 2015-2019 Redistributable Package x64](https://aka.ms/vs/16/release/VC_redist.x64.exe).                                 |                                                                                                                                                                                                                                                                                                                             Required.<br/><br/>The Microsoft Visual C++ Redistributable Package installs runtime components of Visual C++ Libraries required to run applications developed with Visual C++ on a computer that does not have Visual C++ installed.                                                                                                                                                                                                                                                                                                                              |
+|               Microsoft OLE DB Driver for SQL Server                |                                                                                                     Microsoft OLE DB Driver 18.3.0 for SQL Server. [Download Microsoft OLE DB Driver for SQL Server](https://docs.microsoft.com/sql/connect/oledb/download-oledb-driver-for-sql-server?view=sql-server-ver15).                                                                                                     | Required on all BizTalk Server machines. |
```

Comment:
When I view the MD file in browser, a row separator border line is not visible above this row - check?

Reply:
Looks fine to me when i browse the documentation https://github.com/msnehagup/biztalk-docs/blob/pariksg/installandprereq/biztalk/install-and-config-guides/whats-new-in-biztalk-server-2020.md